src/Sylius/Component/Resource/Model/GetIdInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Resource\Model;
+
+/**
+ * @author Joseph Bielawski <stloyd@gmail.com>
+ */
+interface GetIdInterface
```

Comment:
Doctrine calls the standard ID value generation strategy as [`Identity`](http://doctrine-orm.readthedocs.org/en/latest/reference/annotations-reference.html#generatedvalue), so `IdentityInterface`, or `IdentifiableInterface` might be an option. Or simply `ModelInterface`.